services/github/index.ts

```diff
@@ -0,0 +1,46 @@
+import simpleGit, { SimpleGit } from "simple-git";
+import fs from "fs/promises";
+import path from "path";
+import config from "./config.json";
+
+const { owner, repo, ssh } = config;
+
+const { GH_ACCESS_TOKEN } = process.env;
+
+interface WikiParams {
+  wikiPage: string;
+  wikiContent: string;
+}
+
+export default async function updateWiki(params: WikiParams): Promise<void> {
+  try {
+    const git: SimpleGit = simpleGit();
+
+    // Clone the wiki
+    const wikiUrl: string = ssh
+      ? `git@github.com:${owner}/${repo}.wiki.git`
+      : `https://${GH_ACCESS_TOKEN}@github.com/${owner}/${repo}.wiki.git`;
+
+    await git.clone(wikiUrl, `./wikis/${repo}`);
```

Comment:
Something wrong here. I cant upload wiki to my GitHub wiki (I used the github wiki generator as a test wiki). 

Some console output ... and yes i did enter the config data correctly :joy:
![image](https://github.com/Satttoshi/github-wiki-generator/assets/109807794/7572fa28-9b0e-4475-84a7-200f30a0716e)

Reply:
hmm using SSH or HTTPS?

Reply:
SSH

Reply:
think you just did not allow your repository to have a wiki yet, you can do it over the repositories settings.